Description: fix autotools input for compatibility with recent autoconf
 It is not possible to regenerate the blt configure script using current
 versions of autoconf.  Update for recent syntax so that we can get a modern
 (and cross-build-friendly) configure script at build.
Author: Helmut Grohne <helmut@subdivi.de>
Bug-Debian: http://bugs.debian.org/772590

diff -ruN a/aclocal.m4 b/aclocal.m4
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -12,7 +12,7 @@
 cat > conftest.$ac_ext <<EOF
 [#]line __oline__ "configure"
 #include "confdefs.h"
-ifelse(AC_LANG, CPLUSPLUS, [#ifdef __cplusplus
+AC_LANG_CASE([C++], [#ifdef __cplusplus
 extern "C" void exit(int);
 #endif
 ])dnl
diff -ruN a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -127,9 +127,9 @@
 AC_CACHE_VAL(blt_cv_prog_cc, blt_cv_prog_cc=$CC)
 AC_SUBST(CC)
 AC_PROG_CPP
-if test "x${GCC}" != "x" ; then
+AS_IF([test "x${GCC}" != "x"],[
   blt_have_gcc="yes"
-else
+],[
   AC_MSG_CHECKING([if C compiler is really gcc])
     AC_EGREP_CPP(_cc_is_gcc_, [
 #ifdef __GNUC__
@@ -137,7 +137,7 @@
 #endif
 ],      [blt_have_gcc=yes], [blt_have_gcc=no])
   AC_MSG_RESULT([$blt_have_gcc])
-fi
+])
 
 #
 # CFLAGS search order
